tox.ini

```diff
@@ -92,6 +92,38 @@ skip_install = true
 allowlist_externals =
   git
 
+
+[testenv:preview]
+description = preview the docs
+deps =
+  sphinx-autobuild
+  {[testenv:build-docs]deps}
+commands_pre =
+commands =
+  # Retrieve possibly missing commits:
+  -git fetch --unshallow
+  -git fetch --tags
+
+  # Build the html docs with sphinx-autobuild:
+  sphinx-autobuild \
+    -j auto \
+    -b html \
+    -n \
+    -W \
+    . \
```

Comment:
Can we set a `-d` too?

Reply:
As far as I've understood, [-d](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-d) only points to the directory where the parsed source files are stored.

Is this of any relevance, and if so, where should it point to?

Reply:
Yes, it points to the doctree cache. Use `-d "{temp_dir}/.doctrees"` just like others. Ideally, the sphinx commands should make use of mostly the same sets of arguments where possible.